CalculateMethod.java

```diff
@@ -40,6 +40,27 @@ public String calculate(String[] args) {
 			int b = Integer.valueOf(args[1]);
 			return String.valueOf(a % b);
 		}
+	},
+	SIN {
+		@Override
+		public String calculate(String[] args) {
+			Double rad = Double.valueOf(args[0]);
```

Comment:
I see, it seems I haven't updated the remote repo. In my local I'm already using the primitive type. Will resolve all issues. Thanks